ab2d-aggregator/src/main/java/gov/cms/ab2d/aggregator/FileDescriptor.java

```diff
@@ -0,0 +1,17 @@
+package gov.cms.ab2d.aggregator;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.io.File;
+
+/**
+ * Holds the file and it's size. This enables us to manipulate ordering of files by only going to the file
+ * system once
+ */
+@Getter
+@AllArgsConstructor
+public class FileDescriptor {
```

Comment:
It's really not a FileDescriptor.  That has a lot of meaning in the Unix world and doesn't seem to fit here.

How about a FileReferenceHolder?

Reply:
Okay